sdk/core/azure-core-amqp/README.md

```diff
@@ -15,7 +15,7 @@ own AMQP client library that abstracts from the underlying transport library's i
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-core-amqp</artifactId>
-    <version>1.0.0-preview.3</version>
+    <version>1.0.0-preview.5</version>
```

Comment:
this should be preview.4

Reply:
@joshfree  We are releasing 1.0.0-preview.5 next week for azure-core-amqp, thus README should also point to 1.0.0-preview.5 . 1.0.0-preview.4  was released last month.